snuba_sdk/delete_query.py

```diff
@@ -0,0 +1,46 @@
+import json
+from dataclasses import dataclass
+from typing import Any, Dict, List, Union
+
+from snuba_sdk.query import BaseQuery
+
+
+@dataclass(frozen=True)
+class DeleteQuery(BaseQuery):
+    """
+    This represents a snuba delete query.
+    Inputs:
+        storage - the storage to delete from
+        columnConditions - a mapping from column-name to a list of column values
+            that defines the delete conditions. ex:
+            {
+                "id": [1, 2, 3]
+                "status": ["failed"]
+            }
+            represents
+            DELETE FROM ... WHERE id in (1,2,3) AND status='failed'
+    Deletes all rows in the given storage, that satisfy the conditions
+    defined in 'columnConditions'.
+    """
+
+    storage_name: str
+    column_conditions: Dict[str, List[Any]]
```

Comment:
I don't think we should use `Any` here, I think we should explicitly write out the options we allow so far. For now it's just `str` and `int` right?

Reply:
yeah as far as I know just str and int, done